src/example-relay/src/Homepage/locations/LocationsPaginated.js

```diff
@@ -52,7 +52,8 @@ export default function LocationsPaginated(props: Props): Node {
           <Location key={edge?.node?.id} location={edge?.node} />
         ))}
       </LocationList>
-      <Button onClick={loadMore} disabled={!hasNext}>
+      <Button onClick={loadMore} isDisabled={!hasNext}>
+        {/* $FlowFixMe[incompatible-type]: Button children should be FBT */}
```

Comment:
We need to think about this type in SX Design. The goal is to encourage well-built (and therefore translated) applications so maybe we can use here FBT as well. 🤔

Reply:
That tight coupling with FBT?

Reply:
I will relax the `Button` types in a follow-up PR.

Reply:
https://github.com/adeira/universe/pull/2877